.github/pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+# Summary
+
+# Details
+
```

Comment:
Also add some details in this section to specify what it should contain.

```suggestion
# Details

Depends on the nature of the PR :
- **Code changes :** describe the conceptual approach taken in the code and tell the reviewer what part of the changes does what, so that it's easier to read the code
- **Other changes :** Add details if relevant, or remove this section if not useful
```